apps/etterlatte-saksbehandling-ui/client/src/components/behandling/vilkaarsvurdering/Vurdering.tsx

```diff
@@ -123,153 +127,100 @@ export const Vurdering = ({
-            </RadioGroup>
-          </RadioGroupWrapper>
-
-          {VurderingsResultat.IKKE_OPPFYLT === resultat && vilkaar.unntaksvilkaar && vilkaar.unntaksvilkaar.length > 0 && (
-            <>
-              <Unntaksvilkaar>
-                <RadioGroup
-                  legend="Er unntak fra hovedregelen oppfylt?"
-                  size="small"
-                  className="radioGroup"
-                  onChange={(event) => setVilkaarsUnntakType(event)}
-                  value={vilkaarsUnntakType || ''}
-                >
-                  <div className="flex">
-                    {vilkaar.unntaksvilkaar.map((unntakvilkaar) => {
-                      return (
-                        <Radio key={unntakvilkaar.type} value={unntakvilkaar.type}>
-                          {unntakvilkaar.tittel}
-                        </Radio>
-                      )
-                    })}
-                    <Radio key="Nei" value="">
-                      Nei, ingen av unntakene er oppfylt
-                    </Radio>
-                  </div>
-                </RadioGroup>
-              </Unntaksvilkaar>
-            </>
-          )}
-          <VurderingsLabel htmlFor={vilkaar.hovedvilkaar.tittel}>Begrunnelse (obligatorisk)</VurderingsLabel>
-          <Textarea
-            label=""
-            hideLabel={false}
-            placeholder="Gi en begrunnelse for vurderingen"
-            value={kommentar}
-            onChange={(e) => {
-              const kommentarLocal = e.target.value
-              setKommentar(kommentarLocal)
-              kommentarLocal.length >= MIN_KOMMENTAR_LENGDE && setKommentarError(undefined)
-            }}
-            minRows={3}
-            size="small"
-            error={kommentarError ? kommentarError : false}
-            autoComplete="off"
-            name={vilkaar.hovedvilkaar.tittel}
-          />
-          <VurderingKnapper>
-            <Button variant={'primary'} size={'small'} onClick={vilkaarVurdert}>
-              Lagre
-            </Button>
-            <Button variant={'secondary'} size={'small'} onClick={reset}>
-              Avbryt
-            </Button>
-          </VurderingKnapper>
-        </>
-      )}
-      {!vilkaar.vurdering && !aktivVurdering && (
+      {!vilkaar.vurdering && !aktivVurdering ? (
         <IkkeVurdert>
           <Heading size="small">Vilkåret er ikke vurdert</Heading>
           <Button variant={'secondary'} size={'small'} onClick={() => setAktivVurdering(true)}>
             Vurder vilkår
           </Button>
         </IkkeVurdert>
+      ) : (
+        <VurderingsboksWrapper
+          tittel={overskrift()}
+          subtittelKomponent={
+            <VilkaarVurdertInformasjon>
+              <Detail>Manuelt av {vilkaar.vurdering?.saksbehandler}</Detail>
+              <Detail>
+                Sist endret{' '}
+                {vilkaar.vurdering?.tidspunkt ? format(new Date(vilkaar.vurdering.tidspunkt), 'dd.MM.yyyy HH:mm') : '-'}
```

Comment:
har vi ikke noe a.la date-utils for dette? hvis ikke så hade det var fint å ha dette som en generell dateutil-funksjon

Reply:
Flyttet den til utils 👍